4.figures/main_figure_3/main_figure_3.ipynb

```diff
@@ -52,7 +52,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
```

Comment:
prob shouldn't have changed?

Reply:
SEE! lol I am not sure why this keeps happening for me. I will try to be more diligent about it.